mk/spksrc.pre-check.mk

```diff
@@ -1,5 +1,21 @@
 # Common requirement checks
 
+ifneq ($(wildcard BROKEN),)
+  ifneq ($(BUILD_UNSUPPORTED_FILE),)
+    $(shell echo $(date --date=now +"%Y.%m.%d %H:%M:%S") - $(NAME): Broken package >> $(BUILD_UNSUPPORTED_FILE))
+  endif
+  @$(error $(NAME): Broken package)
+endif
+
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
+  ifneq ($(strip $(INSTALLER_SCRIPT)),)
+    ifneq ($(BUILD_UNSUPPORTED_FILE),)
```

Comment:
I don't think you need to check for it here. Is it not covered by the earlier statement?

Reply:
Nvm i don't understand the env var and reviewing this via the iOS GitHub app is a little harder

Reply:
When looking at things naively, every package using `INSTALLER_SCRIPT = src/installer.sh` hasn't been migrate to DSM6+ type default framework method using `SERVICE_SETUP = src/service-setup.sh`.  As such I don't see any value at even trying to build them for DSM7 and therefore they are being excluded from the build process, thus reducing time for the build to occur and removing many false-positive (i.e. packages that are not ready for DSM7 anyway).

There may be another way to address this but I thought this was by far the easiest.

Reply:
@publicarray I've updated the PR narrative to be more explicit.